docs/docusaurus.config.js

```diff
@@ -134,7 +140,7 @@ const config = {
             items: [
               {
                 label: 'Welcome',
-                to: '/docs/overview/introduction',
+                to: '/docs/overview/welcome',
```

Comment:
we should add an entry in _redirects for this path change (to prevent broken links)

Reply:
Yep, already done